DataBase001/Presentor/Mind.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using DataBase001.Model;
+using DataBase001.View;
+using System.Collections.Generic;
+
+
+namespace DataBase001.Presentor
+{
+    
+    public class Mind
+    {
+        Database myBase;
+
+        public void Enter()
+        {
+            while (true)
+            {
+                var MainMenuChoose = CUI.MainMenu();
+
+                switch(MainMenuChoose)
+                {
+                    case 1:
+                        CUI.MenuOfCityes(myBase);                   
+                        break;
+                    case 2:
+                        CUI.MenuOfRotues(myBase);
+                        break;
+                    case 3:
+                        CUI.MenuOfTrains(myBase);
+                        break;
+                    case 0:                        
+                        myBase = BaseTest();
+                        Console.Clear();
+                        Console.WriteLine("Test base loaded");
+                        Console.WriteLine("Press any key to contine");
+                        Console.ReadKey();
+                        break;
+                }
+            }
+            
+
+            
+
+        }
+
+         Database BaseTest()
+        {
+            Database db = new Database();
+
+            db.citys.Add(new City("Moscow", 10000000));
+            db.citys.Add(new City("Paris", 15000000));
+            db.citys.Add(new City("Perm", 1000000));
+            db.citys.Add(new City("Rome", 20000000));
+            db.citys.Add(new City("London", 35000000));
+
+
+            NewRoute("route01", db, "Moscow", "Perm");
+            NewRoute("route02", db, "Moscow", "Paris", "Rome");
+            NewRoute("route03", db, "Perm", "Moscow", "London");
```

Comment:
Это выглядело бы красивее и было бы более единообразно, если бы функция NewRoute просто возвращала новосозданный путь, а добавление его в базу происходило бы тут, тем же способом, что и остальных сущностей
```suggestion
            myBase.routes.Add(NewRoute("route01", db, "Moscow", "Perm"));
            myBase.routes.Add(NewRoute("route02", db, "Moscow", "Paris", "Rome"));
            myBase.routes.Add(NewRoute("route03", db, "Perm", "Moscow", "London"));
```